MyJeopardy.js

```diff
@@ -25,7 +25,7 @@ async function getClues(catId) {
         showing: null // displays nothing
     }));
 
-    return {title: cat.title, clues} // returns the title of the category, and category's clues
+    return {title: category.title, clues} // returns the title of the category, and category's clues
```

Comment:
You need to run `setupStart` before asking me to review next time. This needed to be changed for it to work.

Reply:
Will make the necessary change. And my apologies, I was under the impression that you were checking the accuracy of my code first. But, I understand how difficult that would be, if you don't see that it actually works.

Reply:
No worries! And yeah, usually engineers never submit code reviews until their code is working (i.e. all tests pass).

Reply:
Understood. Makes total sense. So to proceed, should I just do a git pull with my updated/corrected code? Or do I need to follow your pull requests/code review instructions first?

Reply:
You should merge this pull request using github. There should be a big green merge button at the bottom. Then pull down the latest `master` on your computer, and then follow what I said about branching as you make more commits.